IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -126,7 +126,7 @@ - (void)logWithLevel:(MSIDLogLevel)level
             
             NSString *correlationIdStr = @"";
             
-            if (correlationId)
```

Comment:
I think at least we should log a warning about wrong type being passed, if we don't want to assert here :)